### PR TITLE
[GQL-15] UI Kit migration

### PR DESCRIPTION
**Pull request by RichardPickman**
# GraphiQL

[Task](https://github.com/rolling-scopes-school/tasks/blob/master/react/modules/graphiql.md)

## Description

Migrate the project from Flowbite UI kit to Shadcn, due to optimization issues.

image

## Changes Made

Migrate the project from  Flowbite UI Kit to Shadcn/ui

## Related Ticket

[Trello Ticket](https://trello.com/c/kti61Fy0/17-gql-15-migration-from-flowbite-theme-toggler)

## Screenshots (if applicable)

Dark theme:
image

Light theme:
image

### CODE REVIEW

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -1,47 +1,37 @@
...
+      />
+    </Link>
+    <div data-testid="footer-github-names" className="flex items-center gap-2">
+      <FooterGithubLink
+        name="Alisher Mamunov"
+        link="https://github.com/AlanMamphs"
       />
+      <FooterGithubLink name="TODO" link="#" />
+
+      <FooterGithubLink name="TODO" link="#" />
```
Comment (AlanMamphs) [inline src/components/Footer.tsx]: Please add your and Alexander's creds.

**Review by AlanMamphs — CHANGES_REQUESTED** [review]
image

Footer RS logo is also not visible. I believe tests are failing because swcPlugin was removed. I made it conditional in my PR, so you may revert the changes.

File: src/components/Footer.tsx (hunk 1 above)
Comment (RichardPickman) [reply src/components/Footer.tsx]: Working on it!

Header is broken, but that header will be replaced by Animated Header PR

Comment (RichardPickman) [reply src/components/Footer.tsx]: Add my footer link

**Review by AlanMamphs — APPROVED** [review]
LGTM 🥇

**Pull request merged**